tests/unit/delivery-options/deliveryMoment.spec.js

```diff
@@ -0,0 +1,175 @@
+import * as CONFIG from '@/data/keys/configKeys';
+import { MYPARCEL, SENDMYPARCEL } from '@/data/keys/platformKeys';
+import MockDate from 'mockdate';
+import { UPDATED_DELIVERY_OPTIONS } from '@/config/eventConfig';
+import { mockDeliveryOptions } from '@Tests/unit/delivery-options/mockDeliveryOptions';
+import { waitForEvent } from '@Tests/WaitForEvent';
+
+// Timestamps are before all cutoff times
+const FRIDAY = '2020-03-13T10:00:00';
+const SATURDAY = '2020-03-14T10:00:00';
+
+const configMyParcel = {
+  [CONFIG.PLATFORM]: MYPARCEL,
+  [CONFIG.DELIVERY_DAYS_WINDOW]: 7,
+  // Includes Saturday
+  [CONFIG.DROP_OFF_DAYS]: [1, 2, 3, 4, 5, 6],
+  [CONFIG.DROP_OFF_DELAY]: 0,
+  [CONFIG.SATURDAY_CUTOFF_TIME]: '14:30',
+  [CONFIG.ALLOW_MONDAY_DELIVERY]: true,
+};
+
+/**
+ * Delivery days window is always the same for SendMyParcel so it doesn't need to be set.
+ *
+ * @see @/delivery-options/data/request/requestData:9
+ */
+const configSendMyParcel = {
+  [CONFIG.PLATFORM]: SENDMYPARCEL,
+
+  // Includes Friday
+  [CONFIG.DROP_OFF_DAYS]: [1, 2, 3, 4, 5],
+  [CONFIG.DROP_OFF_DELAY]: 0,
+  [CONFIG.FRIDAY_CUTOFF_TIME]: '14:30',
+  [CONFIG.ALLOW_SATURDAY_DELIVERY]: true,
+};
+
+/**
+ * Mock delivery options and returns the array of visible choices for the delivery date option.
+ *
+ * @param {Object} config
+ *
+ * @returns {String[]}
+ */
+async function getMockedDeliveryDateChoices(config) {
+  const wrapper = mockDeliveryOptions({
+    [CONFIG.KEY]: config,
+  });
+  await waitForEvent(UPDATED_DELIVERY_OPTIONS);
+
+  let wrappers;
+  const deliveryDateOption = wrapper.findByTestId('deliveryDate__select__label');
+
+  // When delivery days window is 1 the only option is shown as text instead of a select option.
+  if (deliveryDateOption.exists()) {
+    wrappers = [deliveryDateOption];
+  } else {
+    wrappers = wrapper.findAllByTestId('deliveryDate__select__option').wrappers;
+  }
+
+  return wrappers.map((wrapper) => wrapper.element.getAttribute('data-test-choice'));
+}
+
+describe('Delivery moments', () => {
+  afterEach(() => {
+    MockDate.reset();
+  });
+
+  test('can show monday delivery', async() => {
+    expect.assertions(1);
+    MockDate.set(SATURDAY);
+
+    const choices = await getMockedDeliveryDateChoices(configMyParcel);
+
+    // Monday to Saturday and the Monday after
+    expect(choices).toStrictEqual([
+      '2020-03-16T00:00:00.000Z',
+      '2020-03-17T00:00:00.000Z',
+      '2020-03-18T00:00:00.000Z',
+      '2020-03-19T00:00:00.000Z',
+      '2020-03-20T00:00:00.000Z',
+      '2020-03-21T00:00:00.000Z',
+      // sunday: disallowed
+      '2020-03-23T00:00:00.000Z',
+    ]);
+  });
+
+  it('does not show monday delivery when "allow_monday_delivery" is false', async() => {
```

Comment:
Monday again.

Reply:
A common standard for Jest is keeping all test names fully lowercase. However, I haven't done that in my other files so I'll just capitalize them :)